src/SandboxingLevels.php

```diff
@@ -0,0 +1,180 @@
+<?php
+/**
+ * Class SandboxingLevels.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP;
+
+use AMP_Comments_Sanitizer;
+use AMP_Form_Sanitizer;
+use AMP_Options_Manager;
+use AMP_Script_Sanitizer;
+use AmpProject\AmpWP\Infrastructure\Conditional;
+use AmpProject\AmpWP\Infrastructure\Registerable;
+use AmpProject\AmpWP\Infrastructure\Service;
+
+/**
+ * Experimental service to facilitate flexible AMP.
+ *
+ * @package AmpProject\AmpWP
+ * @since 2.2
+ * @internal
+ */
+final class SandboxingLevels implements Service, Registerable, Conditional {
```

Comment:
Why not call the service just `Sandbox` ? It is being configured via a specific level, of course, but that seems to be a detail, with the fact that we _do have_ a sandbox in place being the most important point.

Reply:
Yeah, that makes sense to drop `Levels`. I think `Sandboxing` works better than `Sandbox`, however.

Reply:
Done in 8e137dbd48b4ba199f561ec29e4558967220e367.